data/monitoring/prom-rule.yaml

```diff
@@ -9,11 +9,9 @@ spec:
   groups:
     - name: kubevirt.cnao.rules
       rules:
+        # +help:summary="Total count of running CNAO operators",type=Gauge
```

Comment:
Please, see comment in my previous review, I believe I have mentioned that I think this is really not a gauge, but a recording rule

Reply:
Yes, it is a recording rule, but in terms of monitoring and user documentation, it is more useful to understand the type of metric created through the use of the recording rules. Because each of those metrics types will have different characteristics and behaviors